packages/git/src/browser/git-view-contribution.ts

```diff
@@ -277,4 +285,42 @@ export class GitViewContribution extends AbstractViewContribution<GitWidget> {
     protected hasMultipleRepositories(): boolean {
         return this.repositoryTracker.allRepositories.length > 1;
     }
+
+    protected updateSyncStatusBarEntry(): void {
+        const entry = this.getStatusBarEntry();
+        if (entry) {
+            this.statusBar.setElement(GitViewContribution.GIT_SYNC_STATUS, {
+                alignment: StatusBarAlignment.LEFT,
+                priority: 100,
+                ...entry
+            });
+        } else {
+            this.statusBar.removeElement(GitViewContribution.GIT_SYNC_STATUS);
+        }
+    }
+    protected getStatusBarEntry(): (Pick<StatusBarEntry, 'text'> & Partial<StatusBarEntry>) | undefined {
+        const status = this.repositoryTracker.selectedRepositoryStatus;
+        if (!status || !status.branch) {
+            return undefined;
+        }
+        if (this.syncService.syncing) {
+            return {
+                text: '$(refresh~spin)',
+                tooltip: 'Synchronizing Changes...'
+            };
+        }
+        const { upstreamBranch, aheadBehind } = status;
+        if (upstreamBranch) {
+            return {
+                text: '$(refresh)' + (aheadBehind ? ` ${aheadBehind.behind}↓ ${aheadBehind.ahead}↑` : ''),
```

Comment:
That is a behavior change, isn't it? Previously we did not show ahead behind if both were 0. I am OK with this, just asking if the change was intentional. The `upstreamBranch` condition is also new.

Reply:
I've mimicked VS Code here. Do you think we should show?

Reply:
Let's stick to VSCode 👍. I just wanted to make sure that the change was made intentionally.

Reply:
Your changes make sense.